### ISSUE TITLE
[9.0]  Long chat conversations, cause bad load performance or browser crash

### ISSUE THREAD

**Issue by dansanti** [odoo/odoo#11863]
Impacted versions: 

9.0c

Steps to reproduce:

1.- Open ad direct message with a user
2.- Make a long conversation with a user ie. 50msg each user
3.- click and go to another module.
4.- click on conversations icon on sup. right area, and select long conversation.
5.- this open a new conversation.
6.- now go to another module ie. accounting and do not close conversation.
7.- now performance decrease and load take long time and some time (ie. firefox) crash , and request stop script.
8.- now close long conversation,  click in another module, ie. account.
9.- performance now is good, and no crash performed

well, this make a bad user experience , specially if you are using Firefox, because open a window asking to stop script.

**Comment by dansanti** [comment]
i thinks, must load an extract, like facebook, or gmail

**Comment by aab-odoo** [comment]
Hello,

I'm not able to reproduce. Anyway, performance shouldn't drop because of an opened chat window because the messages are fetched once, when you open it, not each time you open another application.

Regards

**Comment by dansanti** [comment]
but in a regular machine, happens to when you open odoo, and that chat window is open, cause bad performance and take their time to show all menus, in firefox specially. thanks

**Comment by lukebranch** [comment]
@dansanti ,

Did you have a lot of chat threads open in discuss at the time things were crashing?

What I mean by this is did you have alot of unclosed chat threads from conversations that were finished?

Also, what were your setings in /etc/odoo-server.conf for:

limit_time_cpu
limit_time_real

I have also noticed some issues with performance in discuss, and haven't been able to single out exactly what is going on, however I did notice some issues with keeping too many threads open without clicking the 'x' symbol next to them when I was finished with them.
